### ISSUE TITLE
Licensing Clarification Surrounding InTune and Managing MDE

### ISSUE THREAD

**Issue by jadit1** [MicrosoftDocs/memdocs#3586]
TLDR:
Do you need an InTune license to manage Defender for Endpoint through https://endpoint.microsoft.com?

Scenario:
You have Defender for Endpoint (MDE) deployed to your endpoints. Whether you have Defender for Business, P1 or P2, it doesn't seem to matter. Let's also pretend you don't have any InTune licenses in your tenant.

Let's pretend we run into issues with our ASR rules, or just want manage them. Based on ALL the documentation I've read you should be able to manage the ASR rules through https://endpoint.microsoft.com. The documentation for MDE for Business literally says to go to https://intune.microsoft.com to manage your ASR rules ([LINK](https://learn.microsoft.com/en-us/microsoft-365/security/defender-business/mdb-asr?view=o365-worldwide#enable-your-standard-protection-asr-rules))

`Side note: isn't intune.microsoft.com and endpoint.microsoft.com the exact same thing? Both portals show the same thing.`

However, if you've ever had to call MS Support and asked for help managing the ASR rules through https://endpoint.microsoft.com you are always greeted with: "If you don't have an InTune license you can't use endpoint.microsoft.com to manage MDE. Please use GPO, PowerShell or SCCM."

Based on the chart found here ([LINK](https://learn.microsoft.com/en-us/mem/intune/protect/mde-security-integration#which-solution-should-i-use)) under the "MDE Security configuration" heading and the language found here ([LINK](https://learn.microsoft.com/en-us/mem/intune/protect/mde-security-integration#licensing-and-subscriptions)), a simply license to any version of MDE allows you to manage some of the MDE settings through endpoint.microsoft.com.

Some clarification would be appreciated.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b166b931-f3f7-3323-7288-758cfa5934f1
* Version Independent ID: b166b931-f3f7-3323-7288-758cfa5934f1
* Content: [Use Intune to manage Microsoft Defender for Endpoint Security on devices not enrolled with Microsoft Intune](https://learn.microsoft.com/en-us/mem/intune/protect/mde-security-integration)
* Content Source: [memdocs/intune/protect/mde-security-integration.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/protect/mde-security-integration.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* Technology: ****
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by Brenduns** [comment]
@jadit1  I'm sorry that there has been confusion on this, but your interpretation is correct.  When using the MDE security configuration scenario, you don't need licenses for Intune to manage this configuration via the admin center.  The Microsoft Endpoint Manager admin center is for more than just Intune, and this scenario is one of those - managing devices that don't enroll with Intune but use Microsoft Defender.  Support (in general) is aware of this, though clearly that was not your experience.   This has also been reconfirmed just now with the feature developer owner.

**Comment by jadit1** [comment]
Thanks for the confirmation @Brenduns!

In the future when I'm discussing this with MS Support, can I point them to this thread? Will they believe it?

I'm just asking because I've likely called in about 5 times with MDE issues, and every single time it's been the "You need an InTune license" speech.

**Comment by Brenduns** [comment]
Yes, you can share this thread.  I'm seeking the correct support contacts to reach out to for this to try to get the expectations from the product team sorted with what support is providing. This doesn't sound like it's been an ideal situation.  If your using defender on it's own, then Microsoft Endpoint Manager admin center isn't involved. But for this scenario, using the Microsoft Endpoint Manager admin center to manage defender via this scenario - that is something they should be prepared for.

**Comment by jerryabo** [comment]
@jadit1 can you email your case number(s) to jerryabo@microsoft.com please?

**Comment by jadit1** [comment]
Sorry to be digging this back up again.

Has anything changed in terms of licensing, or is it still working as I initially expected? The only reason I ask is because I'm running into push back again regarding InTune ASR exclusion rules not applying to endpoints that don't have an InTune license.

**Comment by Brenduns** [comment]
[@jadit1](https://github.com/jadit1) There has been no reduction in support, but through a new [opt-in preview released last week](https://learn.microsoft.com/en-us/mem/intune/fundamentals/whats-new#defender-for-endpoint-security-settings-management-enhancements-and-support-for-linux-and-macos-in-public-preview), additional policies are now supported for this scenario (https://learn.microsoft.com/en-us/mem/intune/protect/mde-security-integration?pivots=mdssc-preview) as are some other changes.  For example, with the new preview behaviors, the UI for creating, managing, and assigning these policies is now available from within the Defender Portal which is in addition to the continued support to configure these policies for devices managed *only* by defender via the Intune admin center.  

ASR *Exclusions* are managed via two settings in the profile for *Attack Surface Reduction Rules* and as such would be supported. To date, the development teams for Intune and for Defender have not called out any limitations for some settings that are part of a supported profile.

**Comment by jadit1** [comment]
Thanks for the reply. I doubted anything had changed, but just needed to confirm.

We seem to be seeing a pattern where if the computer is managed by MDE it doesn't get the ASR exclusions. It gets _other_ exclusions, such as Antivirus. 

If the device is InTune managed it gets the ASR exclusions. In the example below, Computers 1, 2 and 3 are getting the ASR exclusions, Computers 4 and 5 don't get the ASR exclusions.
![MDE vs InTune](https://github.com/MicrosoftDocs/memdocs/assets/80353101/befc9330-7554-4b81-a729-84ffa5e856cd)

If we pull up Assignment Status report for the ASR policy, it will show "ASR Only Per Rule Exclusions: Succeeded" for the InTune manager devices, but show "ASR Only Per Rule Exclusions: Not applicable" for MDE managed devices.
![ASR - Not applicable](https://github.com/MicrosoftDocs/memdocs/assets/80353101/a276f5e4-d292-4b1b-b09d-0524c027c0ec)

Any insight would be greatly appreciate. I do have an open support request with Microsoft support, however we are 2 days away from it being a month old...

**Comment by Brenduns** [comment]
@Jadit1  I've forwarded this information to my product group contacts. While those results are not something I've heard about, it's definitely worth sharing directly - even though we can't typically provide troubleshooting support through this docs feedback forum.  Should I hear anything back on this in the next few days, I'll update you here.

**Comment by Brenduns** [comment]
@jadit1   I have an update for you. It appears that _exclusions_ in ASR Rules **_are not_** supported with this scenario after all. I have started a discussion (That remains underway) between Intune and the Defender product teams, and Defender has confirmed that the exclusions will not work for security settings management at this time.   This is new information that was not previously communicated for content, which will be added to the article in next few days, once the developers for both Intune and Defender can confirm for content exactly which settings in this profile, and possibly others, are excluded from support.
